src/routes/userRoute.js

```diff
@@ -1,12 +1,81 @@
 import express from 'express'
 import userController from '../controllers/userController'
 import isAdmin from '../middlewares/isAdmin';
+import Validate from '../middlewares/validator'
 // import auth from '../middlewares/auth'
 
 const router = express.Router();
 
-// userRoute.post('/signin',new userController().login);
-router.post('/', isAdmin, new userController().createUser);
+/**
+ * @swagger
+ * /api/v1/users/:
```

Comment:
It is better to delete unused code.

Reply:
It's a documentation,  I've to change the way to write it.